src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/HologramPlaceholder.java

```diff
@@ -0,0 +1,43 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays;
+
+import lombok.CustomLog;
+import me.filoghost.holographicdisplays.api.placeholder.IndividualPlaceholder;
+import org.betonquest.betonquest.BetonQuest;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Defines HolographicDisplays placeholder <code>{bq:package:variable}</code>.
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.